app/views/users/_form.html.slim

```diff
@@ -0,0 +1,30 @@
+= form_with(model: user) do |form|
+  - if user.errors.any?
+    .alert.label-list
+      h2
+        => pluralize(user.errors.count, "error")
```

Comment:
`=>` adds space at the and of string line
`=` doesn't add 

No need to use two spaces after `|` at the `|  prohibited`